inc/search.class.php

```diff
@@ -3109,8 +3248,9 @@ static function addHaving($LINK, $NOT, $itemtype, $ID, $searchtype, $val) {
     *
     * @return select string
     *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addOrderBy($itemtype, $ID, $order) {
+   public function addOrderBy($itemtype, $ID, $order) {
```

Comment:
This method is used in some plugins (`fusioninventory` for instance), change should be listed in changelog.